### ISSUE TITLE
Go to next error and previous error

### ISSUE THREAD

**Issue by xc1427** [kasecato/vscode-intellij-idea-keybindings#111]
The dual commands can be mapped.

```
    {
        "key": "f2",
        "command": "-editor.action.rename",
        "when": "editorHasRenameProvider && editorTextFocus && !editorReadonly"
    },
    {
        "key": "f2",
        "command": "editor.action.marker.next",
        "when": "editorFocus && !editorReadonly"
    },
    {
        "key": "shift+f2",
        "command": "editor.action.marker.prev",
        "when": "editorFocus && !editorReadonly"
    }
```

**Comment by kasecato** [comment]
Thanks! I released v0.2.22. Please check it out.

**Comment by xc1427** [comment]
FYI, the new released vscode 1.22.2 takes account by default of mapping f2 to problem navigation
